go/obscuronode/enclave/attestation.go

```diff
@@ -16,24 +16,25 @@ type IDData struct {
 
 type AttestationProvider interface {
 	// GetReport returns the verifiable attestation report
-	GetReport(pubKey []byte, owner common.Address) (*obscurocommon.AttestationReport, error)
+	GetReport(pubKey []byte, owner common.Address, hostAddress string) (*obscurocommon.AttestationReport, error)
 	// VerifyReport returns the embedded report data
 	VerifyReport(att *obscurocommon.AttestationReport) ([]byte, error)
 }
 
 type EgoAttestationProvider struct{}
 
-func (e *EgoAttestationProvider) GetReport(pubKey []byte, owner common.Address) (*obscurocommon.AttestationReport, error) {
+func (e *EgoAttestationProvider) GetReport(pubKey []byte, owner common.Address, hostAddress string) (*obscurocommon.AttestationReport, error) {
 	idHash := getIDHash(owner, pubKey)
```

Comment:
The host address needs to be included in that ID hash, were you planning to do that in the follow-up?

Reply:
Fixed